docs/best-practices/documentation.md

```diff
@@ -1,40 +1,51 @@
 # Best Practice: Developer Documentation
 
-## Description
+## Overview
 
 Developer-facing documentation is essential for communicating information about how to begin development on a new project, or how a project is designed to be configured and used.
 This best practice is designed to be a guide for repository-level documentation, i.e., the documentation which would live on the filesystem in the project's source control repository.
 
-## Specific Standards
 
-### README
+## Rationale
+
+Documentation preserves developer knowledge and reduces waste.
+It gives a developers a place to explain how to use their project to other developers and it helps new developers get up and running quickly.
```

Comment:
Probably want to change "It gives a developers...." to "It gives a developer..." or "It gives developers..."

Reply:
@uacate Good catch! Updated.